server/src/main/java/org/opensearch/Version.java

```diff
@@ -92,6 +92,7 @@ public class Version implements Comparable<Version>, ToXContentFragment {
     public static final Version V_2_0_1 = new Version(2000199, org.apache.lucene.util.Version.LUCENE_9_1_0);
     public static final Version V_2_0_2 = new Version(2000299, org.apache.lucene.util.Version.LUCENE_9_1_0);
     public static final Version V_2_1_0 = new Version(2010099, org.apache.lucene.util.Version.LUCENE_9_3_0);
```

Comment:
Not sure if this has any impact, but V_2_1_0 will be using LUCENE_9_2_0 with - https://github.com/opensearch-project/OpenSearch/pull/3676

Reply:
It should. I've fixed it.